### PR TITLE
Collection-activity-toggle-hide-outliers

### LINKED ISSUE #8108: List is not showing on chart but appears in activity
https://canary.kodadot.xyz/ahp/collection/38/activity
image

### LINKED ISSUE #8234: Hide outliers toggle in chart settings
Which charts tells more? I would propose make that checkbox, which could be enabled by default [x] hide outliers and anyone can uncheck it, cc @exezbcz on design

this pr
image

vs canary
image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/8186#issuecomment-1825568520_

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Feature

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #8234
- [x] Closes #8108

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

## Screenshot 📸

- [x] My fix has changed UI

image

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 74bbdff</samp>

Added a feature to hide outliers in the price charts of NFT collections and the global market. Improved the UI and positioning of the dropdown components used to control the chart options. Refactored the `NeoDropdown.vue` component and updated the translation file.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 74bbdff</samp>

> _Sing, O Muse, of the skillful coder who refined the chart_
> _Of the collection's price, with many a clever art._
> _He made a dropdown menu, `NeoDropdown`, shining bright,_
> _And gave it a position prop, to place it left or right._

### CODE REVIEW

File: components/chart/PriceChart.vue [hunk 1]
```diff
@@ -23,6 +23,29 @@
...
+      :mobile-modal="false"
+      class="chart-setting-icon min-width-fit-content"
+      position="bottom-left">
+      <template #trigger>
+        <NeoButton no-shadow variant="icon">
+          <NeoIcon
+            icon="gear"
+            pack="fas"
+            size="medium"
+            :color="isDarkMode ? 'white' : 'black'" />
```
Comment (Jarsen136) [inline components/chart/PriceChart.vue:36]: It looks like no need to add the colour. Or we could try `var(--text-color)`
```suggestion 
/>
```

**Review by Jarsen136 — APPROVED** [review]

File: components/chart/PriceChart.vue (hunk 1 above)
Comment (daiagi) [reply components/chart/PriceChart.vue]: you are right. works just the same without
fixed

**Comment by exezbcz** [comment]
the text to 12px font please

**Review by preschian — APPROVED** [review]

**Comment by daiagi** [comment]
@exezbcz 
all done.
please check again

File: components/chart/PriceChart.vue [hunk 2]
```diff
@@ -23,6 +23,25 @@
       </NeoDropdownItem>
     </NeoDropdown>
 
+    <NeoDropdown
+      :mobile-modal="false"
+      class="chart-setting-icon min-width-fit-content"
+      position="bottom-left">
+      <template #trigger>
+        <NeoButton no-shadow variant="icon">
+          <NeoIcon icon="gear" pack="fas" size="medium" />
```
[the remark this answers is not in the document]
Comment (daiagi) [reply components/chart/PriceChart.vue]: fimga:

image


without pack :
image

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 5.19 USD/DOT ~ 5.78 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0x06038c33ef1abf29b3eb849ff7a261e83d4141e996332c79fb5a54976cb462cb](https://polkadot.subscan.io/extrinsic/0x06038c33ef1abf29b3eb849ff7a261e83d4141e996332c79fb5a54976cb462cb)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**